data/src/main/java/io/homeassistant/companion/android/data/integration/RegisterDeviceRequest.kt

```diff
@@ -0,0 +1,18 @@
+package io.homeassistant.companion.android.data.integration
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import java.util.*
+
+data class RegisterDeviceRequest(
+    var appId: String,
+    var appName: String,
+    var appVersion: String,
+    var deviceName: String,
+    var manufacturer: String,
+    var model: String,
+    var osName: String,
```

Comment:
I do not see the os_name in the doc 
https://developers.home-assistant.io/docs/en/app_integration_setup.html#registering-the-device-1

Reply:
I didn't see it in the docs either, but I got errors when calling so I checked the source: https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/mobile_app/const.py#L106

Need to update the docs.

Reply:
PR for updating the docs:
https://github.com/home-assistant/developers.home-assistant/pull/353